src/apps/LandingRegister/LandingRegister.react.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import {TextField, RaisedButton, IconButton, Colors} from 'material-ui';
+import LinkedStateMixin from 'react-addons-linked-state-mixin';
+
+import LeftBar from './LeftBar.react';
+
+export default React.createClass({
+  displayName: 'LandingRegister',
+
+  mixins: [LinkedStateMixin],
+
+  getInitialState() {
+    return {
+      emailAddress: ''
+    };
+  },
+
+  getStyles() {
+    return {
+      componentBody: {
+        display: 'flex',
+        flexDirection: 'row',
+        justifyContent: 'center',
+        paddingTop: 76
+      },
+      logoBar: {
+        display: 'flex',
+        flexDirection: 'column',
+        maxWidth: 324
+      },
+      contentBar: {
+        display: 'flex',
+        flexDirection: 'column',
+        width: 554,
+        paddingLeft: 38,
+        fontSize: 14
+      },
+      hacklagLogo: {
+        width: 324,
+        height: 324,
+        padding: 20
+      },
+      hacklagLogoDialog: {
+        position: 'relative',
+        left: -76,
+        top: -404,
+        width: 164,
+        height: 114
+      },
+      socialBar: {
+        flexDirection: 'row',
+        justifyContent: 'center',
+        alignItems: 'center',
+        display: 'flex',
+        marginTop: -24
+      },
+      inviteButton: {
+        backgroundColor: '#ec018c',
+        width: '50%'
+      },
+      headlineText: {
+        fontSize: 18
+      },
+      smallerText: {
+        fontSize: 12
+      }
+    };
+  },
+
+  render() {
+    let styles = this.getStyles();
+
+    return (
+      <div style={styles.componentBody}>
+        <LeftBar/>
+        <div style={styles.contentBar}>
+          <div style={styles.headlineText}>
+            We hope to start first <strong><ins>Hackerspace</ins></strong> in <strong>Białystok</strong><br/><br/>
+
+          </div>
+          <p>
+            We are <strong>working hard</strong> to make it real. To be part of this <strong>initiative</strong>
+            or just be <strong>up to date</strong> with the progress please type in your email
+            and we will send you invitation to <strong>our Forum</strong> and <strong>Chat</strong>.<br/><br/>
```

Comment:
our should not be bold, please check also spaces there because I think I saw some missing space somewhere there